pages/index.js

```diff
@@ -9,10 +9,16 @@ import EliminateForm from "@/components/EliminateForm";
 import TombstoneButton from "@/components/TombstoneButton";
 import StyledSection from "@/components/Styles/StyledSection";
 import styled from "styled-components";
+import InteractionMenu from "@/components/InteractionMenu";
+import useLocalStorageState from "use-local-storage-state";
 
 export default function HomePage() {
-  const [selectedPet, setSelectedPet] = useState();
-  const [mode, setMode] = useState("select");
+  const [selectedPet, setSelectedPet] = useLocalStorageState("selectedPet", {
+    defaultValue: {},
+  });
+  const [mode, setMode] = useLocalStorageState("mode", {
+    defaultValue: "select",
+  });
 
   let isDead = false;
```

Comment:
why is this existing, this can not be used as a state, but you could have something like this:

```javascript
const isDead = selectedPet.hp === 0;
```